cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -29,7 +36,7 @@ type CliContext struct {
 func (c *CredentialsCmd) Run(context *CliContext) error {
 	spiffeID, err := spiffeid.FromString(c.SpiffeID)
 	if err != nil {
-		return err
+		return errors.Wrap(err, fmt.Sprintf("failed to parse SPIFFE ID from %s", c.SpiffeID))
 	}
```

Comment:
In addition to the marquee changes, I went through and updated all of the errors so that they're properly messaged and wrapped (wrapping gives us stack traces).